ansible/roles/update/templates/ssmtp.conf.j2

```diff
@@ -0,0 +1,24 @@
+# Managed by Ansible
+# Config file for sSMTP sendmail
+#
+# The person who gets all mail for userids < 1000
+root={{ sysupdate_email }}
+
+# The place where the mail goes. The actual machine name is required no
+# MX records are consulted. Commonly mailhosts are named mail.domain.com
+mailhub={{ ssmtp_mailhub}}:{{ ssmtp_port }}
+
+# Where will the mail seem to come from?
+rewriteDomain={{ ssmtp_rewrite }}
+
+# Username/Password
+AuthMethod=LOGIN
+AuthPass=N7X8vlChzmoh1V4PzUCaA2jglFOA0py4aI3l8Yp2TJGPCIRCVM8PqkyBSvY7yTSmdrU9ye
```

Comment:
This is an accidental p.w. leaka_ge, right? I've reset the password, you should receive it soon

Reply:
Haha :P